.gitignore

```diff
@@ -24,6 +24,8 @@ yarn-error.log*
 
 backend/public
 backend/db
+backend/src/models/db/sql/test/*
+test.db
```

Comment:
was that not dealt with via the original relative .gitignore?

Reply:
not from what I could tell.

Reply:
fair enuf.

Reply:
I just realised there was a second .gitignore file. Is this standard practice?

Reply:
Can be.